doc/sphinx/source/esmvalcore/preprocessor.inc

```diff
@@ -198,100 +390,106 @@ matches the named regions against the requested string.
 
-No depth coordinate is required as this is determined by iris. This
-function works best when the fx_files provide the cell volume.
+No depth coordinate is required as this is determined by Iris. This
+function works best when the fx files provide the cell volume.
 
 See also :func:`esmvalcore.preprocessor.volume_statistics`.
 
 
 3. depth_integration
 --------------------
 
-This function integrate over the depth dimension. This function does a
-weighted sum along the z-coordinate, and removes the z direction of the output
-cube. This preprocessor takes no arguments.
+This function integrates over the depth dimension. It performs a weighted sum
+along the z-coordinate, and removes the z direction of the output cube. It takes no arguments.
 
 See also :func:`esmvalcore.preprocessor.depth_integration`.
 
 
 4. extract_transect
 -------------------
 
-This function extract data along a line of constant latitude or longitude.
-This function takes two arguments, although only one is strictly required.
-The two arguments are `latitude` and `longitude`. One of these arguments
-needs to be set to a float, and the other can then be either ignored or set to
-a minimum or maximum value.
-Ie: If we set latitude to 0 N and leave longitude blank, it would produce a
-cube along the equator. On the other hand, if we set latitude to 0 and then
-set longitude to `[40., 100.]` this will produce a transect of the equator
-in the indian ocean.
+This function extracts data along a line of constant latitude or longitude.
+This function takes two arguments, although only one is strictly required:
+
+* latitude
+* longitude
+
+One of these arguments needs to be set to a float, and the other can then be
+either ignored or set to a minimum or maximum value. For example, if latitude
+is set to 0 and longitude is left blank, the function would produce a cube
+along the equator. If latitude is set to to 0 and longitude to `[40., 100.]` it
+will produce a transect of the equator in the Indian Ocean.
 
 See also :func:`esmvalcore.preprocessor.extract_transect`.
 
 
 5. extract_trajectory
 ---------------------
 
-This function extract data along a specified trajectory.
-The three areguments are: latitudes and longitudes are the coordinates of the
-trajectory.
-
-If two points are provided, the `number_points` argument is used to set a
-the number of places to extract between the two end points.
-
-If more than two points are provided, then
-extract_trajectory will produce a cube which has extrapolated the data
-of the cube to those points, and `number_points` is not needed.
+This function extracts data along a specified trajectory. It requires three
+arguments:
+* latitude_points: list of latitude coordinates
+* longitude_points: list of longiute coordinates
+* number_points: if two points are provided, the `number_points` argument is
+used to set the number of places to extract between the two end points.
```

Comment:
Would that then be done in equal distances? The points set in regular spaces between the two end points?

Reply:
@ledm please comment.

Reply:
Yes, regularly spaced points.